Server/Server.Core/Server.Core.Static/SettingsManager.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "Settings.hpp"
+
+class SettingsManager
+{
+public:
+    explicit SettingsManager(const Settings& settings)
+        : _settings(settings)
+    {}
+
+    std::string GetConnectionOptions() const
+    {
+        return std::string(
```

Comment:
Now, ArgParser doesn't parse arguments with '=' char. After adding improve of the ArgParser, it does work